deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Install Sass
+  gem: name=sass
+       version={{ app_sass_version }}
+       user_install=no
+       state=present
+
+- name: Install Compass
+  gem: name=compass
+       user_install=no
+       state=present
+
+- name: Install bower and gulp
+  command: npm install -g bower gulp
```

Comment:
Without using the npm module for ansible this task keeps failing after running once